### PR TITLE
Added Malware And Phishing Domain Feeds - Bambenek Consulting 

### PR DESCRIPTION
**Pull request by RamboV**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedBambenekConsulting/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Bambenek Consulting Feed
+
+- Added Malware Domains Feed.
+- Added Phishing Domains Feed.
```
Comment (YuvHayun) [inline Packs/FeedBambenekConsulting/ReleaseNotes/1_2_0.md:7]:
```suggestion
- Added support for **Malware Domains** Feed.
- Added support for **Phishing Domains** Feed.
```

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/README.md [hunk 2]
```diff
@@ -36,21 +38,5 @@ Use the Bambenek Consulting feed integration to fetch indicators from the feed.
...
-* High-Confidence C2 All Indicator Feed
-* Sinkhole Feed
-
-List of non-commercial feeds:
-* C2 IP Feed
-* High-Confidence C2 IP Feed
-* C2 Domain Feed
-* High-Confidence C2 Domain Feed
-
-For more information visit [Bambenek Consulting Feeds](https://osint.bambenekconsulting.com/feeds/)
```
Comment (YuvHayun) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/README.md:56]: Why did you remove it?

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @RamboV thx for you contribution.
Looks good. Please see my comments.
Also, please make sure to fill the contribution form.
Note that you're failing on secrets detection (under run validations), if this secrets are fine to sure, please make sure to add them to **.secrets-ignore**

File: Packs/FeedBambenekConsulting/ReleaseNotes/1_2_0.md (hunk 1 above)
Comment (RamboV) [reply Packs/FeedBambenekConsulting/ReleaseNotes/1_2_0.md]: > 

Hello @YuvHayun, updated .

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/README.md (hunk 2 above)
Comment (RamboV) [reply Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/README.md:56]: Hello @YuvHayun , Since all the XSOAR customers of Bambenek Consulting are commercial , hence we removed it.

**Review by YuvHayun — APPROVED** [review]

**Comment by YuvHayun** [comment]
Moving to instance test.

**Review by MLainer1 — APPROVED** [review]
TIM review complete

**Comment by YuvHayun** [comment]
Hey @RamboV, looks good.
Please attach a short demo video where you should the feeds:
two fetch cycles where we can see the first batch of items, the new lastrun set, the incremental fetch of new indicators in the second fetch, and how the indicator looks like in the indicators tab.
One for each new feed.

**Comment by RamboV** [comment]
Hi @YuvHayun, PFA demo video

https://github.com/demisto/content/assets/68921481/792a4f74-94ab-464f-bbf7-4d3df3e10354

**Comment by YuvHayun** [comment]
Looks good, thx for your contribution.
Merging to internal PR.
